services/db/migrations/019_cvm.up.sql

```diff
@@ -0,0 +1,29 @@
+create table `cvm_block`
+(
+    block decimal(65) not null primary key,
+    canonical_serialization varbinary(64000) not null,
+    created_at              timestamp        not null default current_timestamp
+);
+
+create table `cvm_transactions`
```

Comment:
I changed the code so it could happen, but this PR is to capture the atomic import/export tx's..  And these blocks don't have transactions in them, they just have the blockExtraData which are of type evm.UnsignedExportTx, and evm.UnsignedImportTx...

Maybe the table name of evm_transactions is better